scripts/upgrade

```diff
@@ -21,7 +21,7 @@ path_url=$(ynh_app_setting_get --app=$app --key=path)
 is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
 db_user=$db_name
-db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
```

Comment:
Gotta handle the migration for this setting ... if in the past it was called db_pwd, gotta move it to psqlpwd in the upgrade script